GoogleUtilities/AppDelegateSwizzler/GULAppDelegateSwizzler.m

```diff
@@ -39,8 +39,11 @@ typedef void (*GULRealHandleEventsForBackgroundURLSessionIMP)(
     id, SEL, UIApplication *, NSString *, void (^)());
 #pragma clang diagnostic pop
 
```

Comment:
Please add a comment here and at the other `pragma clang diagnostic push` that their purpose is for warning-free code for iOS 7.